core/cas-server-core-services-api/src/test/java/org/apereo/cas/services/DefaultRegisteredServiceAccessStrategyTests.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package org.apereo.cas.services;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Set;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * This is {@link DefaultRegisteredServiceAccessStrategyTests}.
+ * 
+ * @author Martin Böhmer
+ * @since 5.3.8
+ */
+public class DefaultRegisteredServiceAccessStrategyTests {

```

Comment:
This class already exists. Please move the test to https://github.com/apereo/cas/blob/5.3.x/core/cas-server-core-services/src/test/java/org/apereo/cas/services/DefaultRegisteredServiceAccessStrategyTests.java

Reply:
Okay, moved the test. Is it alright now?

Reply:
Yes, all good.